kythe/go/extractors/config/config_parser.go

```diff
@@ -63,7 +63,13 @@ WORKDIR %[2]s
 // the configuration's corresponding repository. Returns the contents of the
 // Dockerfile for the generated composite image. The Dockerfile format is
 // defined here: https://docs.docker.com/engine/reference/builder/
-func NewImage(config *ecpb.ExtractionConfiguration) ([]byte, error) {
+func NewImage(config *ecpb.ExtractionConfiguration, repoDir, outputDir string) ([]byte, error) {
```

Comment:
Please document the new parameters, and in particular what it means if they are empty. (Better yet—since the defaults are exported maybe just let the user pass those)

This might be worth a settings type:

```
type ImageSettings struct {
  RepoDir string
  OutputDir string
}

func NewImage(cfg *ecpb.ExtractionConfiguration, s ImageSettings) ([]byte, error) {
```

That would make the call sites easier to understand. (Though if these are temporary it may not be worth it)